etl/kpinetwork_analizer.py

```diff
@@ -230,39 +393,50 @@ def get_financial_data(
             metric_value = row[index]
             year = years_row[index]
             value = get_metric_value(metric_value)
-            if is_valid_value(metric_value) and value is not None:
 
-                time_period, scenario = get_scenario_data(
-                    row, year, scenario_type, company, scenarios, periods
+            if is_valid_value(metric_value) and value is not None:
+                metric_and_year = scenario_type + "-" + year
+                exist_metric = exists_metric_from_file(
```

Comment:
Change name to existing_metric if it's only one record, existing_metrics for more than one

Reply:
also changed.